docs/source/markdown/options/dns.md

```diff
@@ -12,4 +12,4 @@ host DNS configuration is invalid for the container (e.g., **127.0.0.1**). When
 is the case the **--dns** flag is necessary for every run.
 
 The special value **none** can be specified to disable creation of _/etc/resolv.conf_ in the container by Podman.
-The _/etc/resolv.conf_ file in the image is used without changes.
+The _/etc/resolv.conf_ file in the image is then used without changes.
```

Comment:
Should we add a sentence here explaining that when a network with dns enabled is used that the given nameservers are send to aardvark-dns and not directly inserted into resolv.conf?

I guess that would be the most helpful for users like you?

Reply:
I thought about this too but then didn't want to get too much into detail.  I'll amend the commits and sign off and add something here.

One question though beforehand: What's the deal with the Japanese localization?  Is there a tool to update the files there automatically?

Reply:
> One question though beforehand: What's the deal with the Japanese localization? Is there a tool to update the files there automatically?

I don't know, AFAICT they have been dumped here once (without any long term strategy?) and never really been updated since. I suggest you don't care about them and just don't make any changes to these files. If someone cares about them they can update them later.

Reply:
Added a paragraph that hopefully explains the situation without being too restrictive in its wording to become inaccurate when implementation details change.